payload/payload-100b.data

```diff
@@ -0,0 +1 @@
+fbb0cb87a24408039b4f07c89929b5f619a682ecf61a1d97d495e38f876729ddc5a74c04d2fe9382b5eb85379d0f3027fece
```

Comment:
Barring the fact of leaving the option to user to specify a particular payload file. Couldn't we just create these random contents on the fly?

Reply:
We can but algorithm of generation will be in code. I think that it would be more reasonable to do loading from external files. Users can create there own load it gives more flexibility